icons/pentagon.json

```diff
@@ -1,7 +1,8 @@
 {
   "$schema": "../icon.schema.json",
   "contributors": [
-    "danielbayley"
+    "danielbayley",
+    "jguddas"
```

Comment:
Name shouldn't be here, It's not a design change

Reply:
If you do manual changes/work that should count as a contribution IMO.